pallets/gear/src/lib.rs

```diff
@@ -1658,6 +1665,7 @@ pub mod pallet {
         /// or reply on the message from mailbox.
         #[pallet::call_index(5)]
         #[pallet::weight(<T as Config>::WeightInfo::claim_value())]
+        // MIKITA maybe refund if MessageId is incorrect?
         pub fn claim_value(
```

Comment:
claim_value and send_reply have 2 weight branches: if message contains value and doesn't (should we affect balances storage)!

Reply:
done